uaa/src/main/resources/uaa.yml

```diff
@@ -576,4 +576,10 @@ ldap:
     userDn: 'cn=admin,dc=test,dc=com'
     password: 'password'
     searchBase: 'dc=test,dc=com'
-    searchFilter: 'cn={0}'
\ No newline at end of file
+    searchFilter: 'cn={0}'
+
+# Set a custom content security policy
+# contentSecurityPolicy:
```

Comment:
please remove it from here, examples should be placed in other places, e.g. cargo yml or uaa-release